src/modules/organizations.ts

```diff
@@ -53,7 +47,7 @@ export class Api extends ApiGroup {
     async list(params?: OrganizationListQuery): Promise<Paginated<Organization>> {
         const response = (await this.axios.get<Paginated<IOrganization>>(`/organizations`, { params })).data
 
-        response.data = response.data.map(o => new Organization(o, this.axios))
```

Comment:
I don't think this latest change is correct, you still need to map each of the response items to an `Organization` instance.

Reply:
Mapped out

Reply:
Wait no